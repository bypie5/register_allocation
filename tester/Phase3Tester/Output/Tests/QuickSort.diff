--- SelfTestCases/QuickSort.out	2020-03-05 22:31:29.000000000 -0800
+++ ./Output/Tests/QuickSort.out	2020-03-09 00:14:29.000000000 -0700
@@ -1,22 +1,5 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-9999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+--- Error (ProgramFault) ---
+require function address, found integer10
+  Main() line 20
+  QS.Start() line 39
+  QS.Init() line 341
