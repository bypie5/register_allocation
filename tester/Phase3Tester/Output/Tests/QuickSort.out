--- Error (ProgramFault) ---
require function address, found integer10
  Main() line 20
  QS.Start() line 39
  QS.Init() line 344
